dpypelines/pipeline/shared/transforms/sanity_check.py

```diff
@@ -1,15 +1,40 @@
 from pathlib import Path
 
-import pandas as pd
+
+def sdmx_sanity_check_v1_file_extension(sdmx_file: Path):
+    """
+    Sanity check that the recieved sdmx file is actually an sdmx file.
+    """
+    assert sdmx_file.name.endswith(".xml"), "Invalid sdmx_file"
 
 
 def sdmx_sanity_check_v1(sdmx_file: Path):
     """
     Sanity check that the recieved sdmx file is actually an sdmx file.
     """
+
     assert sdmx_file.name.endswith(".xml"), "Invalid sdmx_file"
 
     try:
-        pd.read_xml(sdmx_file)
+        with open(sdmx_file, "r") as f:
+            f.read()
+        assert sdmx_file.name.endswith(".xml"), "Invalid sdmx_file"
     except Exception as err:
         raise Exception(f"Failed to read in xml - {sdmx_file}") from err
+
+
+# def sdmx_sanity_check_v1(sdmx_file: Path):
```

Comment:
if we're not using it, can we delete it please.

Reply:
Done